roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-cnv-operator.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: openshift-cnv
+
+- name: Create CNV OperatorGroup kubevirt-hyperconverged-group
+  kubernetes.core.k8s:
+    state: present
+
+    definition:
+      apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: kubevirt-hyperconverged-group
+        namespace: openshift-cnv
+      spec:
+        targetNamespaces:
+          - openshift-cnv
+
+- name: Create CNV Subscription
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: hco-operatorhub
+        namespace: openshift-cnv
+      spec:
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        name: kubevirt-hyperconverged
+
+- name: Wait
+  ansible.builtin.pause:
+    seconds: 60
+
+- name: Get Install Plan Name
+  retries: 10
+  delay: 10
+  ansible.builtin.command: oc get subscriptions/hco-operatorhub --namespace openshift-cnv --output=jsonpath='{$.status.installplan.name}'
+  register: __sap_hypervisor_node_preconfigure_register_cnv_subscription_install_plan_name
+  until: __sap_hypervisor_node_preconfigure_register_cnv_subscription_install_plan_name.stdout != ""
+  changed_when: __sap_hypervisor_node_preconfigure_register_cnv_subscription_install_plan_name.stdout != ""
+
+- name: Wait for Install Plan to finish
+  ansible.builtin.command: "oc wait installplan \
+    {{ __sap_hypervisor_node_preconfigure_register_cnv_subscription_install_plan_name.stdout }} --namespace openshift-cnv --for=condition='Installed' --timeout='5m'"
+  register: __sap_hypervisor_node_preconfigure_register_wait_for_installplan
+  changed_when: __sap_hypervisor_node_preconfigure_register_wait_for_installplan.rc != 0
+
+- name: Wait
```

Comment:
Wait above is not sufficient? You could wait for the ready condition of HCO instead of waiting a fixed amount of time.